### ISSUE TITLE
Tile: membership expiration date seems wrong

### ISSUE THREAD

**Issue by c-geek** [duniter/sakia#423]
Hey! Just testing sakia@dev9 and seeing the new tiles:

![image](https://cloud.githubusercontent.com/assets/969136/15207356/d1d83262-1825-11e6-8384-017c01bbf5f1.png)

It says my membership will expire in 22 hours, however this: [/wot/requirements/cgeek](http://cgeek.fr:9330/wot/requirements/cgeek) tells me `14589148` seconds, which means more than 5 months:

```json

{
  "identities": [
    {
      "pubkey": "HnFcSms8jzwngtVomTTnzudZx7SHUQY8sVE1y8yBmULk",
      "uid": "cgeek",
      "meta": {
        "timestamp": "0-E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855"
      },
      "outdistanced": false,
      "certifications": [
        {
          "from": "J78bPUvLjxmjaEkdjxWLeENQtcfXm7iobqB49uT1Bgp3",
          "to": "HnFcSms8jzwngtVomTTnzudZx7SHUQY8sVE1y8yBmULk",
          "expiresIn": 30367948
        },
        {
          "from": "HGYV5C16mrdvE9vpb1S9nMDHkVPsubBgANs9pSb6HWCV",
          "to": "HnFcSms8jzwngtVomTTnzudZx7SHUQY8sVE1y8yBmULk",
          "expiresIn": 30437884
        },
        {
          "from": "BbdyLPyABYzx8Lef3oXzkoiAQ5kn3uU96ZED7Nt17gZx",
          "to": "HnFcSms8jzwngtVomTTnzudZx7SHUQY8sVE1y8yBmULk",
          "expiresIn": 30458892
        },
        {
          "from": "5ocqzyDMMWf1V8bsoNhWb1iNwax1e9M7VTUN6navs8of",
          "to": "HnFcSms8jzwngtVomTTnzudZx7SHUQY8sVE1y8yBmULk",
          "expiresIn": 30915707
        }
      ],
      "membershipPendingExpiresIn": 0,
      "membershipExpiresIn": 14589148
    }
  ]
}
```

How do you compute this expiration date?

**Comment by Insoleet** [comment]
I take the timestamp given in `membershipExpiresIn` and converts it into a remaining time.

I guess Qt takes a timestamp in milliseconds... ?

**Comment by c-geek** [comment]
Hmm not sure, because `14589148 / 1000 = 14589 seconds ` so it should give roughly `4 hours`, and I get 22 hours.

**Comment by Insoleet** [comment]
Or, maybe the toString in the following code should also suggest arguments for months and years. Maybe, instead of converting the remaining months/years to hours, it just cut it from the string. I will try to change it to see what happen...

https://github.com/duniter/sakia/blob/dev/src/sakia/gui/community_tile.py#L112

**Comment by Insoleet** [comment]
Should be fixed in 0.20.0dev10.

**Comment by M5oul** [comment]
Yep, fixed.
